spec/dates_spec.rb

```diff
@@ -5,8 +5,8 @@
 
   it "raises errors for invalid data" do
     expect {
-      Kalculator.evaluate("date(1234)")
-    }.to raise_error(Kalculator::TypeError, "date only works with Strings, got 1234")
+      Kalculator.validate("date(1234)")
```

Comment:
Oh man, this is so cool, i just realized that you don't have to pass in a type environment for expressions with literal values 🎉